presto-product-tests/bin/run_on_docker.sh

```diff
@@ -41,7 +41,12 @@ function run_in_application_runner_container() {
 }
 
 function check_presto() {
-  run_in_application_runner_container /docker/volumes/conf/docker/files/presto-cli.sh --execute "SHOW CATALOGS" | grep -iq hive
+  if [[ "$ENVIRONMENT" == "fileauth" ]]; then
+    password="user123"
+  fi
+  run_in_application_runner_container \
+    bash -c "export PRESTO_PASSWORD=$password;
+    /docker/volumes/conf/docker/files/presto-cli.sh --execute 'SHOW CATALOGS' | grep -iq hive"
```

Comment:
I think PRESTO_PASSWORD should be handled inside `/docker/volumes/conf/docker/files/presto-cli.sh` otherwise things like `./bin/run_presto_cli_on_docker.sh` are not going to work for this environment.

I would add `PRESTO_PASSWORD` env variable to `presto-product-tests/conf/docker/fileauth/docker-compose.yml `

Reply:
there is no need to change `presto-cli.sh`

> I would add PRESTO_PASSWORD env variable to presto-product-tests/conf/docker/fileauth/docker-compose.yml

this should suffice